src/main/hdfs/apps/bi-data-ingestion/lib/01-preprocess_links_app.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash​
+#!/bin/bash
```

Comment:
A special character (200b) has been removed from the end of this line. A 'Shift Enter/Return' in some editors might introduce this character.